charts/catalog/values.yaml

```diff
@@ -4,6 +4,8 @@ image: quay.io/kubernetes-service-catalog/service-catalog:v0.1.31
 # imagePullPolicy for the service-catalog; valid values are "IfNotPresent",
 # "Never", and "Always"
 imagePullPolicy: Always
+# deploymentStrategy for service-catalog; value values are "RollingUpdate" and "Recreate"
```

Comment:
Have we documented both the deployment strategies and the differences somewhere?

Reply:
@shubheksha: It's a kube term, https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy